macros/plugins/snowflake/create_external_table.sql

```diff
@@ -26,12 +26,14 @@
         {%- endfor -%}{%- endif -%}
         {%- if not infer_schema -%}
             {%- for column in columns %}
+                {%- set column_alias = column.alias if column.alias else column.name %}
+                {%- set column_alias_quoted = adapter.quote(column_alias) if column.quote else column_alias %}
                 {%- set column_quoted = adapter.quote(column.name) if column.quote else column.name %}
```

Comment:
Is this line needed or used anymore?
```suggestion
```

Reply:
it is needed as the column_quoted is used in the expression part of the definition, we can't use column_alias_quoted in value: , hope that clarifies the need.